### PR TITLE
Use git system to update image

### PR DESCRIPTION
**Pull request by Barakmor1**

### CODE REVIEW

File: github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml [hunk 1]
```diff
@@ -1,5 +1,39 @@
...
+        - image: quay.io/kubevirtci/bootstrap:v20230626-e1b7af2
+          env:
+            - name: DOCKER_PREFIX
+              value: quay.io/kubevirt
+          command:
+            - "/usr/local/bin/runner.sh"
+            - "/bin/sh"
+            - "-c"
+            - |
+              cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io &&
```
Comment (iholder101) [inline github/ci/prow-deploy/files/jobs/kubevirt/managed-tenant-quota/managed-tenant-quota-presubmits-1.1.yaml:28]:
```suggestion
              docker login --username $(cat "$QUAY_USER") --password="$QUAY_PASSWORD" quay.io &&
```

**Comment by Barakmor1** [comment]
rehearsal-push-release-mtq-images

**Comment by Barakmor1** [comment]
rehearsal-push-release-mtq-images

**Comment by Barakmor1** [comment]
rehearsal-push-release-mtq-images

**Comment by Barakmor1** [comment]
rehearsal-push-release-mtq-images

**Pull request closed without merging**